fpmsyncd/routesync.cpp

```diff
@@ -747,6 +754,79 @@ void RouteSync::onRouteMsg(int nlmsg_type, struct nl_object *obj, char *vrf)
 }
 
 /* 
+ * Handle label route
+ * @arg nlmsg_type      Netlink message type
+ * @arg obj             Netlink object
+ */
+void RouteSync::onLabelRouteMsg(int nlmsg_type, struct nl_object *obj)
+{
+    struct rtnl_route *route_obj = (struct rtnl_route *)obj;
+    struct nl_addr *daddr;
+    char destaddr[MAX_ADDR_SIZE + 1] = {0};
+
+    daddr = rtnl_route_get_dst(route_obj);
+    nl_addr2str(daddr, destaddr, MAX_ADDR_SIZE);
+    SWSS_LOG_INFO("Receive new route message dest addr: %s", destaddr);
```

Comment:
"new route" -> "new label route".   Would be easier for debugging

Reply:
onRouteMsg is used for IP routes, not label routes.

Reply:
This API is for Label routes only.

Reply:
@smahesh You are correct.  I will fix it.

Reply:
@smahesh This is fixed.